server/opts.go

```diff
@@ -9,3 +11,13 @@ func Verbose() Opt {
 		s.verbose = true
 	}
 }
+
+// Favicon makes the server serves given file at /favicon.ico
+func Favicon(path string) Opt {
+	if _, err := os.Stat(path); err != nil {
+		panic(err)
+	}
+	return func(s *Server) {
+		s.faviconPath = path
+	}
```

Comment:
```suggestion
func Favicon(path string) (Opt, error) {
	if _, err := os.Stat(path); err != nil {
		return nil, err
	}
	return func(s *Server) {
		s.faviconPath = path
	}, nil
```
We should bubble this error up, seems like something `main` should give more information about.